docs/infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+Student Robotics has a variety of infrastrcture in order to support its mission.
+This document aims to provide an outline of the general structure.
+
+Currently our infrastructure as a whole does not have a designed owner, instead
+individual pieces are looked after by volunteer Teams or just by volunteers who
+have historically maintained them.
```

Comment:
Might be worth referencing the Infrastructure Team proposal here, or waiting until that proposal is finalised.

Reply:
I'd honestly hoped that that proposal would be sorted by now. Until it does appear I'm mixed on whether mentioning it is a good idea. If it doesn't appear soon, then over time it would become much less useful for this to mention it.

Reply:
1026b4d acknowledges the infra team, however I'm deliberately omitting a description of who's responsible for what for now. I think we'll want to work out a bit more generally where in the runbook to put "team X is responsible for Y" type information as right now there isn't one obvious home for it.